pom.xml

```diff
@@ -294,9 +294,14 @@
             <version>1.7</version> <!-- Or 1.8-SNAPSHOT -->
         </dependency>
         <dependency>
-            <groupId>org.hibernate</groupId>
+            <groupId>org.hibernate.validator</groupId>
             <artifactId>hibernate-validator</artifactId>
-            <version>5.0.3.Final</version>
+            <scope>compile</scope>
```

Comment:
This should remain `provided` or be deleted, as `compile` is the default.

Reply:
IMHO you should take a look at this:

![image](https://user-images.githubusercontent.com/5468333/91275956-22458400-e781-11ea-87db-dd7c8f955ce2.png)

We're using an **internal** validator class in our code and I'm not sure if this is a supported use case. This should be refactored and use [Java 8 Collections API](https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html).

Reply:
BTW @kcondon I noticed you're testing on Payara 5.201. It's not important for this issue (both 5.201 up to 5.2020.4 use Hibernate Validator 6.1.2 Final), but we should make sure that testers, QA etc all use the same version referenced in `pom.xml`.

Reply:
@poikilotherm - I suspected that this was the reason I couldn't use provided, but refactoring that code is a separate issue IMHO